bazel/python_utils/BUILD.bazel

```diff
@@ -16,3 +16,9 @@ py_library(
     srcs = ["coverage_decorator.py"],
     visibility = ["//visibility:public"],
 )
+
+py_library(
+    name = "conftest",
+    srcs = ["conftest.py"],
+    visibility = ["//visibility:public"],
```

Comment:
Can we restrict this to `bazel/python_test.bzl`?

Reply:
I'm not sure, the target above this one has the same visibility and used only in `bazel/python_test.bzl`.

Reply:
Can you try and see if it is possible?

Reply:
Not possible. The file is added to every test, therefore it has to be visible from every test.

Reply:
Then let's keep it public.